python/kserve/kserve/protocol/rest/openai_datamodels.py

```diff
@@ -0,0 +1,109 @@
+import time
+from typing import Dict, List, Literal, Optional, Union
+
+from pydantic import BaseModel, Field
+
+from vllm.utils import random_uuid
```

Comment:
do we need vllm imports here?

Reply:
We don't. I'll clean it up once `OpenAIModel()` implementation is finished.